.github/workflows/python-app.yml

```
@@ -25,9 +29,14 @@ jobs:
         rm -rf /opt/hostedtoolcache
         df -h
     - name: Set up Python 3.11
-      uses: actions/setup-python@v5
+      uses: actions/setup-python@v3
```

actionlint (1.7.9)

the runner of "actions/setup-python@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)